### ISSUE TITLE
Support C Takumi x Oboro not work

### ISSUE THREAD

**Issue by Nicaiah** [TeamIf/FireEmblemIf-FanTranslation#15]
In Invisible Kingdom the supports don't work.
Thanks for your work !

**Comment by Nicaiah** [comment]
Sorry I just look this won't be fixed...

A question : I actually have some supports of Festival DLC translated, but I don't have experience for patch in game. Can I share this supports somewhere for you ?

**Comment by ShadySheikah** [comment]
Thanks for letting us know! I recall this support being brought up in another issue report, but it was marked as fixed. Can you give us some more information regarding your issue with the supports in Invisible Kingdom? Which version of the patch are you playing? Is it just Takumi and Oboro? Is the conversation still in Japanese, or do the supports simply not play?

Regarding DLC, we won't be working on any of them until the main game is close to complete. However, if you'd like to share what you have with us, we'd be glad to implement it when the time comes. I'm reachable on Twitter and the Serenes Forest forum under the same username.

**Comment by Nicaiah** [comment]
For the support : I actually play with the last patch 4.1.  Actually it's only for Support C of Takumi x Oboro the support is not play.  And the Support B actually WORKS for information (nott tested A and S).

For DLC : Ok for the moment I will be patient, and when I see you are finish the main game, I will be back for share this ! 

Thanks ! I follow you !

**Comment by jftm999** [comment]
this issue was addressed by me in the topic **"Hoshido story and Support Typo"**, **galaxygamr98**  replied that it was fixed along with other missing dialog conversations, but it needs to be implanted in the next patch

**Comment by Danielosama** [comment]
@ShadySheikah 
I am playing the European Special Edition with your latest patch, and during the Hoshidan route, the C support between Takumi and Oboro doesn't play at all.

**Comment by jftm999** [comment]
I'm playing right now with the latest patch and it's working fine, did you update the Repository files in your SD card, they update it every month

**Comment by Danielosama** [comment]
@jftm999 
Repository files? I am afraid to say I don't know what you are referring to :x

**Comment by jftm999** [comment]
Sorry about last post. See in the first link on the page of the translation https://serenesforest.net/forums/index.php?/topic/67613-fire-emblem-if-fan-translation-under-new-management/

It says GitHub Repository.  When you download the translation files. There is a file called patch. Before you apply the translation,  go download the repository files and copy them and replace them in the patch file. Since the translation files last updated in February and the Repository updated last month.

**Comment by Danielosama** [comment]
@jftm999 
Yes, I figured what you were talking about a bit later. I patched again my game with the latest commit and Takumi x Oboro C Support works now. However, it's Marx x Aqua S Support that doesn't play now x(
